lib/auto-migrations/private/run-alter-strategy/private/inform-re-failed-alter-stratagem.js

```diff
@@ -114,10 +114,8 @@ module.exports = function informReFailedAlterStratagem(err, operationName, model
   message +=
   '\n'+
   '-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- \n'+
-  'Some development data was recovered (see `'+ relPathToLogFile + '`).  '+
   'Any existing `'+ modelIdentity + '` records were deleted, but your data from OTHER models '+
   '(including any relationships tracked in foreign keys and join tables) might still be intact.  '+
-  'If you care about recovering any of that data, be sure to back it up now before you continue.\n'+
```

Comment:
I think this line can stay (because you still might back up the data that's intact)